packages/react-core/src/components/Page/Page.tsx

```diff
@@ -287,7 +288,7 @@ class Page extends React.Component<PageProps, PageState> {
           )
         )}
       >
-        {isBreadcrumbWidthLimited ? <div className={css(styles.pageMainBody)}>{breadcrumb}</div> : breadcrumb}
+        {isBreadcrumbWidthLimited ? <PageMainBody>{breadcrumb}</PageMainBody> : breadcrumb}
```

Comment:
Since `<PageMainBody>` should always wrap the children of all page section (`__main-group`), was there a reason it's only being added here with `isBreadcrumbWidthLimited`?

Though looking at this branch, looks like you're on core [alpha.150](https://github.com/thatblindgeye/patternfly-react/blob/980971bab884928888780067e629538d8902eb47/packages/react-styles/package.json#L22) but the core changes to handle the changes to allow for multiple bodies are in [alpha.162](https://github.com/patternfly/patternfly/pull/6766) - can you bump core in this PR?

Reply:
Bumped to latest Core, this will fail due to other PRs needing to go in first (Switch, JumpLinks, etc).